Inwoong 4

in-woong opened the pull request:
- prettier 적용
- Footer 만들기
- Slider 오류 수정

in-woong committed:
hotfix: prettier 적용

in-woong committed:
hotfix: footer 추가

oinochoe reviewed (commented):
prettier 작업하시느라 수고하셨습니다.

이제 코드 정리도 어느정도 되었으니, 최적화도 해보세요.  😄 

keywords : useCallback, useMemo, React.memo

in-woong committed:
refactor: 코드 최적화

1. React.memo 이용 props 변화에따른 리렌더링 방지(slider, signup login)
2. useCallback이용 함수 재생성 방지(slider ,signup, login)

in-woong commented:
Slider, Login, Signup Component 에 대해 최적화를 위해 React.memo, useCallback을 사용해 보았습니다.

oinochoe reviewed (commented):
자주 사용되는 함수에 대해서 useCallback으로 기본 함수에 대한 연산을 반복하지 않게 함으로 인해서 hilighting 도구에 있어서 성능이 최적화 된 것을 직접 디버깅 할 수 있습니다 😄

in-woong committed:
Merge branch 'develop_v1' into inwoong_4
